<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="49065" started-at="2018-01-26T05:03:20Z" finished-at="2018-01-26T05:04:09Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="49065" started-at="2018-01-26T05:03:20Z" finished-at="2018-01-26T05:04:09Z">
      <class name="c2info_OrderBuk_SupportCancelledTCs.TC_001_VerifyInvalidPrescFromTBV">
        <test-method status="PASS" signature="setup()[pri:0, instance:c2info_OrderBuk_SupportCancelledTCs.TC_001_VerifyInvalidPrescFromTBV@176d53b2]" name="setup" is-config="true" duration-ms="16612" started-at="2018-01-26T10:33:20Z" finished-at="2018-01-26T10:33:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:c2info_OrderBuk_SupportCancelledTCs.TC_001_VerifyInvalidPrescFromTBV@176d53b2]" name="beforeMethod" is-config="true" duration-ms="12" started-at="2018-01-26T10:33:37Z" finished-at="2018-01-26T10:33:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void c2info_OrderBuk_SupportCancelledTCs.TC_001_VerifyInvalidPrescFromTBV.verifyInvalidPrescFromTBV() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="verifyInvalidPrescFromTBV()[pri:0, instance:c2info_OrderBuk_SupportCancelledTCs.TC_001_VerifyInvalidPrescFromTBV@176d53b2]" name="verifyInvalidPrescFromTBV" duration-ms="31295" started-at="2018-01-26T10:33:37Z" finished-at="2018-01-26T10:34:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyInvalidPrescFromTBV -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:c2info_OrderBuk_SupportCancelledTCs.TC_001_VerifyInvalidPrescFromTBV@176d53b2]" name="afterMethod" is-config="true" duration-ms="1" started-at="2018-01-26T10:34:08Z" finished-at="2018-01-26T10:34:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyInvalidPrescFromTBV status=SUCCESS method=TC_001_VerifyInvalidPrescFromTBV.verifyInvalidPrescFromTBV()[pri:0, instance:c2info_OrderBuk_SupportCancelledTCs.TC_001_VerifyInvalidPrescFromTBV@176d53b2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:c2info_OrderBuk_SupportCancelledTCs.TC_001_VerifyInvalidPrescFromTBV@176d53b2]" name="endTest" is-config="true" duration-ms="1093" started-at="2018-01-26T10:34:08Z" finished-at="2018-01-26T10:34:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- c2info_OrderBuk_SupportCancelledTCs.TC_001_VerifyInvalidPrescFromTBV -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
